### PR TITLE
Add release blog post for 1.8 release

### PR DESCRIPTION
**Pull request by nak3**
As per title, this patch adds the blog post for 1.8 release.

/cc @lionelvillard @snneji @abrennan89
/cc @davidhadas @upodroid (I added about security guard & image signing)

### CODE REVIEW

File: blog/docs/releases/announcing-knative-v1-8-release.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+Follow the instructions in [Installing Knative](https://knative.dev/docs/install/) to install the components you require.
+
+This release brings a number of smaller improvements to the core Knative Serving and Eventing components, and several improvements to specific plugins.
+
+## Table of Contents
+- [General](#general)
+- [Serving](#serving)
+- [Eventing](#eventing)
+- [`kn` CLI](#client)
```
Comment (lance) [inline blog/docs/releases/announcing-knative-v1-8-release.md:17]: Can we get `func` CLI in here too?

File: blog/docs/releases/announcing-knative-v1-8-release.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+[Release Notes](https://github.com/knative/client/releases/tag/knative-v1.8.0)
+
+#### 💫 New Features & Changes
+
+* Add `--scale-activation` flag to `service create` command options (#1729, @vyasgun)
+* Provide cli options to enable Kubernetes user, uid, and group impersonation via `--as`, `--as-group` and `--as-uid` flags (#1745, @a7i)
+
+### Bug or Regression
+
+* Fix release script version calculation (#1737, @dsimansk)
```
Comment (lance) [inline blog/docs/releases/announcing-knative-v1-8-release.md:90]:
```suggestion
* Fix release script version calculation (#1737, @dsimansk)

## Functions

This is the first release for Functions as a part of Knative Core.

[Release Notes](https://github.com/knative/func/releases/tag/knative-v1.8.0)

#### 💫 New Features & Changes

* Cancel pipeline run on SIGINT/SIGTERM (#1329, @matejvasek)
* On cluster build using direct source upload (i.e. git is not needed) (#1298, @matejvasek)
* Changes package name from knative.dev/kn-plugin-func to knative.dev/func (#1311, @lance)

```

File: blog/docs/releases/announcing-knative-v1-8-release.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+- [Serving](#serving)
+- [Eventing](#eventing)
+- [`kn` CLI](#client)
+- [Knative Operator](#operator)
+
+## General
+
+### 🚨 Breaking or Notable
+
+* All of our images are now signed with cosign.
```
Comment (upodroid) [inline blog/docs/releases/announcing-knative-v1-8-release.md:24]:
```suggestion
```

I didn't get image signing out in time. But `func` and `kn` are notarised.

**Comment by nak3** [comment]
Thank you! Updated.

**Comment by csantanapr** [comment]
thanks @nak3 
/unhold

**Pull request merged**